util/libcrypto.num

```diff
@@ -5425,3 +5425,4 @@ ASN1_item_d2i_ex                        5552	3_0_0	EXIST::FUNCTION:
 ASN1_TIME_print_ex                      5553	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_get0_provider                  5554	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_CTX_get0_provider              5555	3_0_0	EXIST::FUNCTION:
+BN_consttime_randomized_swap            ?	3_0_0	EXIST::FUNCTION:
```

Comment:
Does this need to be a public function?

Reply:
Does it need to be a separate function?

This potentially sets as up for another situation similar to the consttime flag, where we/contributors might forget to use the randomized version instead of the normal version.

Wouldn't it be preferable to unconditionally use the version resistant to Nonce@Once proposed here and the previous PRs? I doubt there are spots where we are using `BN_consttime_swap()` where we do not want to be protected from attacks like the Nonce@Once one.

Reply:
Agreed, better to do this all of the time.